src/futures.ts

```diff
@@ -270,6 +262,14 @@ export function handlePositionLiquidated(event: PositionLiquidatedEvent): void {
     positionEntity.isLiquidated = true;
     positionEntity.save();
   }
+  if (tradeEntity) {
+    tradeEntity.size = event.params.size.times(tradeEntity.size);
```

Comment:
maybe a comment here would help too

```suggestion
    // size will be either -1 or 1 in this scenario (see handlePositionModified for details)
    tradeEntity.size = event.params.size.times(tradeEntity.size);
```